ansible_collections/arista/avd/docs/how-to/first-project.md

```diff
@@ -452,7 +449,7 @@ servers:
   DCI_RTR01:
     rack: DCI
     adapters:
-      - server_ports: [Eth1, Eth2]
+      - endpoint_ports: [Eth1, Eth2]
         switch_ports: [Ethernet5, Ethernet5 ]
```

Comment:
Delete las space between Ethernet5 and ] to make it consistent with the whole document.